tasks/dataverse-apache.yaml

```diff
@@ -1,21 +1,21 @@
+---
+# dataverse/tasks/dataverse-apache.yaml
+
 - name: install Apache for RedHat/CentOS
   yum: name="{{item}}" state=latest
   with_items:
     - httpd
     - mod_ssl
   when: ansible_os_family == "RedHat" and
         ansible_distribution_major_version == "7"
-  tags: apache
```

Comment:
Previously, all individual tasks within a file were tagged like this. I've simply moved the tags attribute to the `include: …` block of the main tasks file. A benefit of this is that you don't have to remember to tag new tasks should you add them. (In fact, a few existing tasks were not tagged, so this is essentially a bugfix.)